locales/en-US/experiments.ftl

```diff
@@ -54,7 +76,7 @@ nomore404sSubtitle = Powered by the Wayback Machine
 nomore404sDescription = Tired of dead ends on the web? We'll let you know when there's a saved version of what you're looking for on the Internet Archive's Wayback Machine.
 nomore404sIntroduction = <p>This one is simple. If you hit a 404 page while browsing the web, we will let you know if the content you're looking for is archived on the Internet Archive's Wayback Machine.</p>
 nomore404sMeasurements0 = We collect basic usage on how many times you encounter a Page Not Found error (code 404), how many times a cached version of that page exists from Archive.org, and how many times you choose to view the cached version.
-nomore404sMeasurements1 = To provide cached versions of pages, we send 404 error page URLs to Archive.org. <a href="https://archive.org/" target="_blank">Archive.org</a> discloses its privacy policy <a href="https://archive.org/about/terms.php" target="_blank">here</a>.
+nomore404sMeasurements1 = To provide cached versions of pages, we send 404 error page URLs to Archive.org. <a href="https://archive.org/" target="_blank">Archive.org</a> discloses its privacy policy <a href="https://archive.org/about/terms.php" target="_blank">here</a>.</li>
```

Comment:
Good catch.  Please whack it on https://github.com/mozilla/testpilot/blob/master/content-src/experiments/no-more-404s.yaml#L29